media/css/firefox/family/components/_hero.scss

```diff
@@ -26,31 +26,64 @@
         margin-top: $layout-lg;
     }
 
-    // todo: try grid, confirm minmax sizing on lockup
     @supports (flex: 1 0 auto) {
-        $gap: $layout-lg;
-        $half-width: 40%;
-        @media #{$mq-md} {
-            .l-flex {
-                display: flex;
-                flex-wrap: wrap;
-                gap: $gap;
-                align-items: center;
-                justify-content: center;
-            }
+        $gap: $spacing-lg;
+        $half-width: 50%;
+
+        .l-grid {
+            display: flex;
+            flex-wrap: wrap;
+            gap: $gap;
+            align-items: center;
+            justify-content: center;
 
             h1 {
                 flex: 1 0 $half-width;
-                margin-bottom: 0;
+                display: flex;
+                justify-content: center;
             }
 
             .c-card {
                 flex: 0 1 $half-width;
             }
 
             .c-blurb {
-                flex: 0 0 auto;
+                flex: 0 1 auto;
+                text-align: center;
+            }
+
+            @media #{$mq-md} {
+                $gap: $layout-lg;
+            }
+        }
+    }
+
+    @supports (display: grid) {
+        $gap: $layout-lg;
+        $half-width: 40%;
+
+        @media (min-width: 1195px) {
+            .l-grid {
+                display: grid;
+                justify-content: space-between;
+                column-gap: $gap;
+                row-gap: $gap;
```

Comment:
**nit**: if we remove the row-gap (or set it to 0), we can keep the default blurb margin and remove the (pre-existing) margin-top override for .c-blurb on L83
```suggestion
```